clients/redshift/merge.go

```diff
@@ -83,7 +84,7 @@ func (s *Store) Merge(ctx context.Context, tableData *optimization.TableData) er
 			continue
 		}
 
-		err = utils.BackfillColumn(ctx, s, col, fqName)
+		err = utils.BackfillColumn(*config.FromContext(ctx).Config, s, col, fqName)
 		if err != nil {
```

Comment:
Can we inject the Store object with `config` and just pass it around when necessary?

Reply:
You mean pass store to `utils.BackfillColumn`?

Reply:
And also `TableData`?

Reply:
Sorry for not being clear.

I meant have `config` as an object on store, such that this function becomes something like:

```go
err = utils.BackfillColumn(s.Config, s, col, fqName)
```

Reply:
Yep, will do that in a follow up.